template.yaml

```diff
@@ -84,6 +84,11 @@ Conditions:
           - !Ref Environment
           - "dev"
 
+  IsDevelopment:
```

Comment:
Please use the existing IsnotDevelopment and use NOT, some examples here: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-conditions.html